### DISCUSSION TITLE
javax.script in TeaVM

### DISCUSSION THREAD

**Discussion by Quillraven** [xpenatan/gdx-teavm#118] [Q&A]
Hi,

I tried to introduce [ktx-script](https://github.com/libktx/ktx/tree/master/script) in my project which uses javax.script.ScriptEngineManager in the background and seems like this is not working in TeaVM because of:

```
| ERROR[0]
| Class: javax/script/ScriptEngineManager.java:222
| Method: javax.script.ScriptEngineManager.getEngineBy(Ljava/lang/String;Ljava/util/Map;Ljava/util/function/Function;)Ljavax/script/ScriptEngine;
| Text: Method java.util.stream.Stream.ofNullable(Ljava/lang/Object;)Ljava/util/stream/Stream; was not found
```

From what I saw Stream.ofNullable is not implemented in TeaVM? Is there a way to still get it to work? Or is there an alternative scripting support?

Also, the main page on github is showing broken badges for the versions ;)

Thank you for any help!

**Comment by Quillraven** [comment]
seems to be a tough/impossible task: https://github.com/konsoletyper/teavm/issues/930

--> nevermind :)

**Reply by xpenatan** [reply]
I see. The only scripting language I have working is lua.
